scalalib/src/publish/Pom.scala

```diff
@@ -30,7 +30,8 @@ object Pom {
       artifact: Artifact,
       dependencies: Agg[Dependency],
       name: String,
-      pomSettings: PomSettings
+      pomSettings: PomSettings,
+      versionScheme: VersionScheme
```

Comment:
This change breaks the binary interface of the method `apply`, is this OK?

Reply:
Here I would rather like to see a more generic `properties: Map[String, String]` parameter. The logic to calc the version scheme should happen on caller side.

In general, I'd say we want to keep a binary compatible overload, but as we are looking to get the next 0.10.x milestone out which isn't binary compatible anyways, we might get away with it. Maybe, we want provide a default value, then the change is source compatible.